.tekton/build-pipeline-must-gather.yaml

```diff
@@ -98,6 +98,10 @@ spec:
       set of values is determined by the configuration of the multi-platform-controller.
     name: build-platforms
     type: array
+  - default: "false"
```

Comment:
The commit contains disabling and enabling two different checks, please align the commit title and rephrase the description, and remove double sign-off